shared/admin-tools/user/admin.ts

```diff
@@ -202,9 +211,8 @@ export const createDawsonUser = async ({
     },
   };
 
-  const url = deployingColorUrl ?? `https://api.${EFCMS_DOMAIN}/users`;
   try {
-    await axios.post(url, user, headers);
+    await axios.post(deployingColorUrl, user, headers);
```

Comment:
Huh, why not build the deploying color url here with the `process.env.DEPLOYING_COLOR`? I suppose that refactoring can be considered down the road. It just seems strange to rely on an entire URL that's passed into the function to create a user in the system. I realize you didn't make that change to begin with.

Reply:
We agree with you but it seems out of scope for this story.

We originally touched this as we were trying to remove the uncolored-api but reverted that change. We just updated the signature of this function to make it a little more strict for how you call it.